packages/phone-number-privacy/signer/src/common/io.ts

```diff
@@ -26,7 +26,8 @@ export abstract class IO<R extends OdisRequest> {
   abstract authenticate(
     request: Request<{}, {}, R>,
     warnings?: string[],
-    logger?: Logger
+    logger?: Logger,
+    timeoutMs?: number
```

Comment:
Same as above, doesn't look like we need to add this parameter here since we're never passing it in to the implementations

Reply:
Yes, basically the signer includes warnings in its response to the combiner so that the combiner knows when signers are failing open/closed or having other difficulties. This allows us to have a window into signer behavior from monitoring the combiner even though we don't operate all the signers. We don't return these warnings to the user though, so when the combiner fails open/closed on authentication we don't need to store it in warnings